examples/data-objects/todo/src/Todo/TodoListView.tsx

```diff
@@ -0,0 +1,122 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { CollaborativeInput } from "@fluid-example/example-utils";
+import { SharedString, type ISharedString } from "@fluidframework/sequence/legacy";
+import { Tree, type TreeNode } from "@fluidframework/tree/legacy";
+import React, { useEffect, useRef, useState } from "react";
+
+import { TodoItemView } from "../TodoItem/index.js";
+
+// eslint-disable-next-line import/no-unassigned-import
+import "./style.css";
+
+import { type TodoListDataObject } from "./index.js";
+
+export interface TodoViewProps {
+	readonly todoModel: TodoListDataObject;
+	readonly getDirectLink: (itemId: string) => string;
+}
+
+// TODO: This was copied over from the "@fluid-experimental/tree-react-api" package.
+// This should be imported from that package, once it is no longer experimental.
+export function useTree(subtreeRoot: TreeNode): void {
```

Comment:
Since this is used by both `TodoListView` and `TodoItemView`, can we move it to a shared location somewhere? Ideally not under either the `Todo` or `TodoItem` directories.